tests/test_accumulatorMeanStack.py

```diff
@@ -0,0 +1,202 @@
+import unittest
+
+import numpy as np
+import numpy.testing as testing
+
+import lsst.utils.tests
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+from lsst.meas.algorithms import AccumulatorMeanStack, stats_ctrl_to_threshold_dict
+
+
+class AccumulatorMeanStackTestCase(lsst.utils.tests.TestCase):
+    def make_test_images_to_coadd(self):
+        """Make test images to coadd."""
+        np.random.seed(12345)
+
+        n_image = 10
```

Comment:
Perhaps indicate in a comment that `n_image` has to be at least 5 and the `image_size` has to be at least 150 pixels along each dimension.

Reply:
There's no such restriction, this is just arbitrary.

Reply:
But you do explicitly use first 5 images to introduce artifacts by hand and if I later edit the test code to `n_image = 3`, the test would fail.

Reply:
Ah, I think I see the confusion.  The idea is to set either 1 bad pixel in the stack or half in the stack to test different scenarios with the thresholding.  And the number of images and the setting of the images in the tests are not independent.  I've added the following comment:
```
        # Choose an arbitrary number of images and image size.
        # Below we set bad pixels for either 1 or half these images,
        # at arbitrary locations.
```